.github/workflows/ci-build.yaml

```
@@ -416,14 +416,17 @@ jobs:
       - run: yarn --cwd ui lint
       - run: yarn --cwd ui deduplicate
       # Deduplicate UI deps for dependabot PRs
-      - if: github.actor == 'dependabot[bot]'
+      - name: Commit deduplicated yarn.lock for Dependabot PRs
+        if: github.actor == 'dependabot[bot]'
         run: |
           git config --global user.name 'github-actions[bot]'
           git config --global user.email 'github-actions[bot]@users.noreply.github.com'
           git add ui/yarn.lock
-          git commit -s -m 'chore: deduplicate yarn.lock'
+          git commit -s --alow-empty -m 'chore: deduplicate yarn.lock'
```

```suggestion
          git commit -s --allow-empty -m 'chore: deduplicate yarn.lock'
```

I had a typo here, dang it. I actually re-typed this when testing locally, and didn't have a typo that time.
This is causing another CI failure in https://github.com/argoproj/argo-workflows/pull/12887#issuecomment-2038223808, will fix this shortly

---

Fixed the typo in #12889